integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -147,9 +147,9 @@ worker:
     requests:
       cpu: "1"
       memory: "2G"
-    ports:
-      rpc: 29999
-      web: 30000
+  ports:
+    rpc: 29999
+    web: 30000
```

Comment:
`ports` field moved up the tree. This is because we rely on the existence of `master/worker.resources` to exist to generate the resource request/limit block. However this `master/worker.resources` can exist and only contain ports, which messes up the logic.

Also in theory, it's better to have flatter structure whenever possible.